### DISCUSSION TITLE
XFORMERS conflict VLAD A1111

### DISCUSSION THREAD

**Discussion by Torcelllo** [vladmandic/sdnext#1781] [Q&A]
I start my VLAD A1111 using --medvram --backend diffusers.
Time after time, it worked perfectly, Then an Xformers conflict ...

![VLAD Xformers Fail](https://github.com/vladmandic/automatic/assets/114806454/1e3c6057-abbc-4f4b-9290-a3a598ac66eb)
[Xformers Fault LOG.txt](https://github.com/vladmandic/automatic/files/12116499/Xformers.Fault.LOG.txt)

 Any ideas? When it works, VLAD A1111 is excellent!

**Comment by vladmandic** [comment]
i've added extra check around that specific setting, fixed in dev, will publish later today.
